Baekjoon/l-lyun/11047.py

```diff
@@ -0,0 +1,27 @@
+'''
+작성자: l-lyun
+문제 링크: https://www.acmicpc.net/problem/11047
+카테고리: 그리디
+
+k == coin일 때 누락
+현재 조건문 k >= coin에서, 기존 답안은 등호 명시 안해줘서 오답 발생
+'''
+import sys
+
+n, k = map(int, sys.stdin.readline().split())
+coins = []
+
+for i in range(n):
+    coin = int(sys.stdin.readline())
+    coins.append(coin)
+
+coins.sort(reverse=True)
+result = 0
+for coin in coins:
+    cnt = 0
+    if (k >= coin):
+        cnt = k // coin
+        k -= cnt * coin
+    result += cnt
```

Comment:
```suggestion
    if k >= coin:
        cnt = k // coin
        k -= cnt * coin
        result += cnt
```

1. 파이썬에서 if 문 뒤 괄호는 불필요합니다.
2. if 조건에 부합하면 cnt가 result에 더해지기 때문에 cnt = 0 으로 초기화 후 항상 result += cnt 하는 구문 대신 if 조건문에서 모두 해결 가능할 것 갔습니다.